javascript/code-challenges/challenges-01.test.js

```diff
@@ -154,7 +160,30 @@ Return the resulting output array.
 ------------------------------------------------------------------------------------------------ */
 
 const fizzbuzz = (arr) => {
-  // Solution code here...
+
+  const newArray = [];
+
+  arr.forEach(number => {
+
```

Comment:
Really great work here!

The only feedback I have for you is regarding spacing. For the if else statements, you could remove a lot of this extra space and indent the block up to be directly under the forEach.